spec/controllers/api/v2/data/sessions_controller_spec.rb

```diff
@@ -1,14 +1,12 @@
 require 'rails_helper'
 
 describe Api::V2::Data::SessionsController do
-  describe "GET 'last'" do
-    before do
+  describe "GET last" do
+    it 'returns id' do
       FactoryGirl.create(:mobile_session)
```

Comment:
I'm for 2. This is a quote from the authors of the gem which I agree with

> My general advice, though, is to avoid Factory Girl as much as is reasonably possible. Not because it’s bad or unreliable software (Factory Girl is very reliable; we’ve used it successfully since 2008), but because its inherent persistence mechanism is calling #save! on the object, which will always take longer than not persisting data.
(https://thoughtbot.com/blog/speed-up-tests-by-selectively-avoiding-factory-girl)

Reply:
esattamente